### PR TITLE
Refactoring - Move CefCookie conversion to TypeConversion

### PR DESCRIPTION
**Pull request by merceyz**
- Moved conversion of `CefCookie` -> `Cookie` to `CefSharp.Core\Internals\TypeConversion.h`
- Copied `ConvertCefTimeToDateTime` from `CefSharp.BrowserSubprocess.Core\TypeUtils.h` to `CefSharp.Core\Internals\TypeConversion.h` as including the file caused errors

The conversion of cookies was duplicated in
- `CefSharp.Core\Internals\ClientAdapter.cpp`
- `CefSharp.Core\Internals\CookieVisitor.cpp`
- `CefSharp.Core\ResourceHandlerWrapper.cpp`

### CODE REVIEW

File: CefSharp.Core/ResourceHandlerWrapper.cpp [hunk 1]
```diff
@@ -64,49 +65,6 @@ namespace CefSharp
...
-                lastAccess.second,
-                lastAccess.millisecond);
-        }
-
-        return cookie;
+        return TypeConversion::FromNative(cefCookie);
     }
-}
\ No newline at end of file
+}
```
Comment (amaitland) [inline CefSharp.Core/ResourceHandlerWrapper.cpp]: I though all the line endings would have been normalised as part of #2504?

Comment (merceyz) [reply CefSharp.Core/ResourceHandlerWrapper.cpp]: Yeah, it should have.
For some stupid reason Visual Studio doesn't apply the `insert_final_newline` rule unless you open all files and save them manually.
I'll write a quick program to fix this.

Comment (amaitland) [reply CefSharp.Core/ResourceHandlerWrapper.cpp]: Only if you really want to. Fine to leave as is.

Comment (merceyz) [reply CefSharp.Core/ResourceHandlerWrapper.cpp]: So you're fine with this PR getting merged?

Comment (merceyz) [reply CefSharp.Core/ResourceHandlerWrapper.cpp]: > Only if you really want to. Fine to leave as is.

Rather not have these sorts of changes show up in unrelated commits/PR in the future

**Pull request merged**